packages/sources/avalanche-platform/schemas/env.json

```diff
@@ -0,0 +1,19 @@
+{
+  "$id": "https://external-adapters.chainlinklabs.com/schemas/example-source-adapter.json",
+  "title": "Example Source Adapter",
+  "description": "Example Source Adapter (this title and description were pulled from source/schemas/env.json)",
```

Comment:
Update

Reply:
Good catches! I've updated all references to the example adapter in the latest commit